Assets/MixedRealityToolkit/_Core/Boundary/MixedRealityBoundarySystem.cs

```diff
@@ -136,11 +251,7 @@ private void SetTrackingSpace()
         /// </remarks>
         private void SetPlatformBoundaryVisibility()
```

Comment:
If we're setting shouldn't that also mean we're passing a parameter?

Reply:
i'll rename to update

Reply:
removed complete.y instead